text/src/autogluon/text/automm/utils.py

```diff
@@ -1232,3 +1233,31 @@ def turn_on_off_feature_column_info(
                 per_model_processor.requires_column_info = flag
 
     return data_processors
+
+def get_mixup(
+        config: DictConfig,
+        output_shape: int,
+):
```

Comment:
Add document here. Also, refer to the mixup/cutmix paper and the implementation in TIMM in the documentation